twingate/internal/provider/datasource/remote-network.go

```diff
@@ -44,6 +44,11 @@ func RemoteNetwork() *schema.Resource {
 				Description:  "The name of the Remote Network",
 				ExactlyOneOf: []string{"id"},
 			},
+			"location": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The location of the Remote Network",
```

Comment:
@alexmensch please check description

Reply:
We should list out the possible options here. So let's change it to: `The location of the Remote Network. Must be one of the following: AWS, AZURE, GOOGLE_CLOUD, ON_PREMISE, OTHER`

Reply:
The only downside to this is that the docs aren't automatically up to date with our API, but I don't think these are going to change very often.